src/lib/get-node-without-tc.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * The contents of this file are subject to the Common Public Attribution License Version 1.0 (the “License”); you may not use this file except in compliance with the License. You may obtain a copy of the License at https://mpapp-public.gitlab.io/manuscripts-frontend/LICENSE. The License is based on the Mozilla Public License Version 1.1 but Sections 14 and 15 have been added to cover use of software over a computer network and provide for limited attribution for the Original Developer. In addition, Exhibit A has been modified to be consistent with Exhibit B.
+ *
+ * Software distributed under the License is distributed on an “AS IS” basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the specific language governing rights and limitations under the License.
+ *
+ * The Original Code is manuscripts-frontend.
+ *
+ * The Original Developer is the Initial Developer. The Initial Developer of the Original Code is Atypon Systems LLC.
+ *
+ * All portions of the code written by Atypon Systems LLC are Copyright (c) 2019 Atypon Systems LLC. All Rights Reserved.
+ */
+
+import { TrackedAttrs } from '@manuscripts/track-changes-plugin'
+import { JSONNode, ManuscriptNode } from '@manuscripts/transform'
+
+export function checkSuggestionsFor(
+  node: JSONNode,
+  onNodeReal: (node: JSONNode) => void
+) {
+  let textString = ''
+  if (node.content?.length) {
+    const newContent = []
+
+    nodesLoop: for (let i = 0; i < node.content?.length; i++) {
```

Comment:
can we use node descendants, and build text when going through title node

Reply:
I'm using this approach here because it allows not only to get textContent without TC but to recursively get nodes without track changes, which is a more generic case we will need for other tasks. If'll use descendants I can only retrieve text, since I cannot modify the nodes as it will affect the actual document.